File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Python/Prenotazione.py
class Prenotazione:

    def __init__(self, codiceCampo, ora) -> None:
        self.codiceCampo = codiceCampo
        self.ora = ora

    def __str__(self) -> str:
        return "(" + str(self.codiceCampo) + "," + str(self.ora) + ")"

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Java/Prenotazione.java
package Esame24Gennaio2020.Java;

public class Prenotazione {
    
    protected int codiceCampo, ora;

    public Prenotazione(int codiceCampo, int ora) {
        this.codiceCampo = codiceCampo;
        this.ora = ora;
    }

    @Override
    public String toString() {
        return "(" + codiceCampo + "," + ora + ")";
    }
}
